scripts/deploy_manager.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Why we need this script here?

Reply:
accidentally pushed it in this patch.
that's for the upcoming tests for the tm repo